components/eam/src/chemistry/utils/tracer_data.F90

```diff
@@ -1851,6 +1997,10 @@ subroutine interpolate_trcdata( state, flds, file, pbuf2d )
                    do k = 1,file%nlev
                       pin(:,k) = file%levs(k)
                    enddo
+                   !!Currently designed for linoz_v3 use only
```

Comment:
Should this comment be updated as Linoz v2 is also supported now?

Reply:
Updated.